_tools/release/03_release.ts

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env -S deno run --allow-read --allow-write --allow-env --allow-net --allow-run=git --no-check
+#!/usr/bin/env -S deno run --allow-read --allow-write --allow-env --allow-net --allow-run=git,deno --no-check
```

Comment:
I'm curious where `deno` is used. (`--allow-run=deno` is almost close to `--allow-all`, so ideally we want to avoid this..

Reply:
It's actually not used, but I just added it to match the other script. I removed it now.

I think we should essentially treat this code as `--allow-all` anyway though. Right now we're only importing from our own repos and transitively octokit on skypack. I should vendor that actually. I will open a PR in deno_automation.